.github/workflows/dotnet.yml

```diff
@@ -0,0 +1,32 @@
+# This workflow will build a .NET project
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-net
+
+name: Ensure ASP.NET Project Builds
+
+on:
+  push:
+    branches: [ "dev", "main" ]
```

Comment:
It may not be necessary to add `main` to the push event because pushing to main it blocked via GitHub Branch protection rules. However, it won't harm anything having main listed under the push event.